### ISSUE TITLE
Why is the loss combination of CrossEntropy and Dice used?

### ISSUE THREAD

**Issue by lferlings** [milesial/Pytorch-UNet#342]
Hey, thanks for your great work on this project milesial. Could you (or someone seeing this) please explain to me why a combination of both the standard cross entropy loss and the dice loss is used?
Thank you

**Comment by Blupblupblup** [comment]
Such a combination is mentioned in this paper https://arxiv.org/pdf/2006.14822.pdf :

```
J. Combo Loss
Combo loss [15] is defined as a weighted sum of Dice
loss and a modified cross entropy. It attempts to leverage the
flexibility of Dice loss of class imbalance and at same time
use cross-entropy for curve smoothing.
```

Maybe some learning curve smoothing related motivation, according to https://stats.stackexchange.com/a/344403/241224 ?

**Comment by milesial** [comment]
Above answers are correct
